sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Breaking changes
 - Renamed `Id` for `Identity` in all the `StartRecognizeIdDocuments` functionalities. For example, the name of the method is now `StartRecognizeIdentityDocuments`.
+- Rename  the model `ReadingOrder` to `FormReadingOrder`.
```

Comment:
Should it be "Renamed" like the entry above? Also, nit: double space after "Rename".

Reply:
fixed